test/cuda/metrics.jl

```diff
@@ -66,7 +66,7 @@
             end
         end
 
-        verts = get_verts_packed(m)
+        verts = get_verts_packed(m) |> cpu
```

Comment:
Why do we bring the to cpu?

Reply:
These are tests; the latter test computation involved Array, which caused trouble